tools/cockpit.spec

```diff
@@ -165,6 +178,11 @@ exec 2>&1
 
 make -j4 %{?extra_flags} all
 
+%if 0%{?with_selinux}
+    make -f /usr/share/selinux/devel/Makefile cockpit.pp
```

Comment:
Interesting, that looks into the selinux/ folder by default and takes all files there? That seems quite implicit.

Reply:
> that looks into the selinux/ folder by default and takes all files there? 

Indeed.